ui/scss/component/_membership.scss

```diff
@@ -66,3 +66,8 @@
 
   margin-bottom: 21px;
 }
+
+.getPaymentCard {
+  font-size: 19px;
+  margin-bottom: 10px;
+}
```

Comment:
_Just a discussion at this point, although I don't see why existing variables can't be used._

----
Spoke with Jessop a bit on this...
- [ ] The amount of hardcoded css values that's coming up lately is really bugging me.
    - Not using `var(--font-size-*)` maybe explains why the GUI today doesn't have consistent font sizes (e.g. tags no longer have the consistent look-and-feel -- they are sometimes as big as buttons, other times small as it should be).
    - Not using `var(--spacing-*)` makes the component spacing looks incoherent with the rest of the theme.  Why such a specific value like 223px and 21px?

    But we've also created some counter-arguments ourselves:
    - Not enough variable 
    - Don't have an overall architecture and design scheme to fully reap the benefits of variables.
    - Does it work for responsive layout design/auto rescaling?

- [ ] `getPaymentCard` probably fails the BEM guideline
    - I've been at fault too because I didn't know about it, but now trying to fix/practice moving forward.
    
- [ ] Inline styles like `<h2 style={{ marginBottom: '15px' }}>` breaks separation-of-concerns.
    - There should be a theme-coherent className to achieve what you want, and if not, create one so it's re-usable?

Reply:
- [ ] I don't think `.button--close` is used.